com.unity.shadergraph/Editor/Current/Data/Nodes/Artistic/Adjustment/Levels.shadersubgraph

```diff
@@ -505,10 +505,10 @@
     "m_Type": "UnityEditor.ShaderGraph.DynamicVectorMaterialSlot",
     "m_ObjectId": "2b3ce540db09401ca88a09830c5fb5fd",
     "m_Id": 1,
-    "m_DisplayName": "B",
+    "m_DisplayName": "Exp",
```

Comment:
Are you updating these names to keep consistency between current and GTF versions?

Reply:
This is a subgraph node made in old Shader Graph.  Tracy updated the names of the ports in the old shader graph Power node, so when I opened this subgraph (which contains a power node), this was updated to match.  I like the names you used in your first sample node much better than what we used to have, so we went with it.  :)